src/Our.Umbraco.InnerContent/Web/Controllers/InnerContentApiController.cs

```diff
@@ -32,18 +32,19 @@ public IEnumerable<object> GetAllContentTypes()
         [HttpGet]
         public IEnumerable<object> GetContentTypesByGuid([ModelBinder] Guid[] guids)
         {
-            return Services.ContentTypeService.GetAllContentTypes()
-                .Where(x => guids == null || guids.Contains(x.Key))
-                .OrderBy(x => x.SortOrder)
-                .Select(x => new
-                {
-                    id = x.Id,
-                    guid = x.Key,
-                    name = x.Name,
-                    alias = x.Alias,
-                    icon = string.IsNullOrWhiteSpace(x.Icon) || x.Icon == ".sprTreeFolder" ? "icon-folder" : x.Icon,
-                    tabs = x.CompositionPropertyGroups.Select(y => y.Name).Distinct()
-                });
+            var contentTypes = Services.ContentTypeService.GetAllContentTypes(guids).OrderBy(x => x.SortOrder).ToList();
+            var blueprints = Services.ContentService.GetBlueprintsForContentTypes(contentTypes.Select(x => x.Id).ToArray()).ToArray();
+
+            // NOTE: Using an anonymous class, as the `ContentTypeBasic` type is heavier than what we need (for our requirements)
+            return contentTypes.Select(ct => new
+            {
+                name = ct.Name, // TODO: localize the name (in case of dictionary items), e.g. `localizedTextService.UmbracoDictionaryTranslate`
+                description = ct.Description, // TODO: localize the description (in case of dictionary items), e.g. `localizedTextService.UmbracoDictionaryTranslate`
+                key = ct.Key,
+                icon = string.IsNullOrWhiteSpace(ct.Icon) || ct.Icon == ".sprTreeFolder" ? "icon-document" : ct.Icon,
+                blueprints = blueprints.Where(bp => bp.ContentTypeId == ct.Id).ToDictionary(bp => bp.Id, bp => bp.Name)
+                // TODO: tabs = ct.CompositionPropertyGroups.Select(y => y.Name).Distinct()
```

Comment:
@mattbrailsford On this line, I can't find anywhere that references the `tabs` field (from the call to `GetContentTypesByGuid`).  It's probably a breaking-change to remove it, but what are your thoughts?

Reply:
hmm, are tabs fetched later on for the dialog window then?

Reply:
Yeah, they come back with the scaffold object.

Reply:
Ahh, ok, then yea, looks like they aren't needed at this point then. Technically another breaking change though yea, but again, I doubt anyone is using it :)